opensearch-search-quality-evaluation-plugin/src/main/java/org/opensearch/eval/runners/SearchMetrics.java

```diff
@@ -8,6 +8,55 @@
  */
 package org.opensearch.eval.runners;
 
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Provides the ability to calculate search metrics and stores them.
+ */
 public class SearchMetrics {
```

Comment:
Let's make the _k_ configurable and start with the set of three metrics.

If we can make this easily extensible that would open the door to contributions.